builtins/amp-ad.js

```diff
@@ -283,8 +283,17 @@ export function installAd(win) {
      * @private
      */
     noContentHandler_() {
-      this.element.removeChild(this.iframe_);
-      this.toggleFallback(true);
+      if (!this.fallback_) {
+        this./*REVIEW*/changeHeightIfAllowed(0, () => {
+          this.deferMutate(() => this.element.style.display = 'none');
```

Comment:
As Malte requested - he wanted to do it in the same mutate context. If so - let's do.